ccd-definition/ComplexTypes.json

```diff
@@ -1556,5 +1556,307 @@
     "FieldType": "Document",
     "ElementLabel": "Upload a file",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Telephone",
+    "ListElementCode": "telephoneNumber",
+    "FieldType": "Text",
+    "ElementLabel": "Telephone number",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Telephone",
+    "ListElementCode": "contactDirection",
+    "FieldType": "Text",
+    "ElementLabel": "Contact direction",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Telephone",
+    "ListElementCode": "telephoneUsageType",
+    "FieldType": "Text",
+    "ElementLabel": "Telephone usage type",
+    "SecurityClassification": "Public"
+  },
```

Comment:
why have these been added?

Reply:
Ideal is we add 0 new fields to UI

Reply:
@swalker125 so is the idea, partyType, telephone, email etc be removed from the ccd definition and data migration will do the rest?

Reply:
so there is an update to the message. If you look at my branch I annotate my classes with @JsonIgnore. This means that null values do not get instantiated. So I do not need to add fields that I do not want. You will need to add fields like partyType and hide them as we want these to populated in an AboutToSubmit callback